frontend/src/Page/Packages.elm

```diff
@@ -486,10 +486,10 @@ viewResultItem nixosChannels channel showInstallDetails show item =
                   Maybe.withDefault d m
 
               showTeamEntry githubTeam =
-                  (a [ href ((String.append "https://github.com/orgs/NixOS/teams/") githubTeam) ] [ text githubTeam ])
+                  (a [ href ((String.append "https://github.com/orgs/NixOS/teams/") githubTeam) ] [ text ("@NixOS/" ++ githubTeam) ])
           in
           li []
-              ([ text team.shortName ] ++ (List.map showTeamEntry (maybe team.githubTeams [])))
+              ([ text (team.shortName ++ ": ") ] ++ (List.map showTeamEntry (maybe team.githubTeams [])))
```

Comment:
I think this would look a bit weird for teams that have no mapping to GitHub teams (e.g. [gitlab-container-registry](https://search.nixos.org/packages?channel=unstable&show=gitlab-container-registry&from=0&size=50&sort=relevance&type=packages&query=gitlab-container-registry), AFAICS) - could you tweak that?